applications/nrf_desktop/src/hw_interface/leds.c

```diff
@@ -19,11 +19,11 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_LED_LOG_LEVEL);
 
+#define LED_ID(led) ((led) - &leds[0])
```

Comment:
The same macro definded in two source files, wouldn't it be better to avoid code duplication?

Reply:
I was thinking about it, but this is only one macro and `leds` does not refer to the same variable in both cases. I would prefer to leave it as it is.

Reply:
Well there is always some point when something starts to exist. And macro is like function, e.g. add(a,b) doesn't mean it must always add one particular variable.

Reply:
Currently i would have to create one special header for this macro. I'm not sure if this is necessary (this would add additional dependency).